platform/viewer/src/routes/WorkList/WorkList.jsx

```diff
@@ -194,6 +194,20 @@ function WorkList({ history, data: studies, isLoadingData, dataSource }) {
     });
   };
 
+  useEffect(() => {
+    if (studies && studies.length && studies.length < 101) {
+      const defaults = defaultFilterValues;
+      const { sortBy, sortDirection } = filterValues;
+      if (sortBy === defaults.sortBy && sortDirection === defaults.sortDirection) {
+        setFilterValues(filterValues => ({
+          ...filterValues,
+          sortBy: 'studyDate',
+          sortDirection: 'descending'
+        }));
+      }
+    }
+  }, [studies, filterValues]);
```

Comment:
I think this will actually cause "Clear Filters" button to always exist. We just need to make sure the data/studies passed into this component are sorted by default if it was requested with no filter values.

There is this method in `extensions/default/src/DicomWebDataSource/qido.js`

```
/**
 *
 * @param {object} dicomWebClient - Client similar to what's provided by `dicomweb-client` library
 * @param {function} dicomWebClient.searchForStudies -
 * @param {string} [studyInstanceUid]
 * @param {string} [seriesInstanceUid]
 * @param {string} [queryParamaters]
 * @returns {Promise<results>} - Promise that resolves results
 */
function search(
  dicomWebClient,
  studyInstanceUid,
  seriesInstanceUid,
  queryParameters
) {
  // Studies
  return dicomWebClient.searchForStudies({
    studyInstanceUid: undefined,
    queryParams: queryParameters,
  });
}
```

I think if the above criteria is met, you would apply a client-side sort to the result set here.